lib/grizzly/commands/command.ex

```diff
@@ -296,14 +296,16 @@ defmodule Grizzly.Commands.Command do
 
   defp calculate_rssi_values(stats) do
     average =
-      Keyword.get(stats, :rssi_hops)
+      Keyword.get(stats, :rssi_hops, [])
       |> Enum.filter(fn rssi -> rssi != :not_available end)
       |> calculate_average()
 
     Keyword.put(stats, :rssi_dbm, average)
     |> Keyword.put(:rssi_4bars, calculate_bars(average))
   end
 
+  # Average RSSI DBM is 0 when no data. So 0 bars.
+  defp calculate_bars(0), do: 0
```

Comment:
0 dbm is a really strong signal so this seems weird. Can no data be a small number? Perhaps -125 dbm like LTE?

Reply:
Or a smaller negative number.